arrow/src/main/java/org/apache/iceberg/arrow/vectorized/VectorHolder.java

```diff
@@ -131,4 +140,10 @@ public Object getConstant() {
     }
   }
 
+  public static class PositionVectorHolder extends VectorHolder {
```

Comment:
Seems like technically this class is redundant since the user can use VectorHolder directly, but is probably good for readability.

Reply:
Yes, I added a private `VectorHolder` constructor for `PositionVectionHolder` which I don't want others to use it directly.